turn_by_turn/xtrack_line.py

```diff
@@ -85,30 +106,26 @@ def convert_to_tbt(xline: Line) -> TbtData:
         )
     npart = npart_set.pop()
 
-    # Precompute masks for each monitor and particle_id (Half the time to compute this bit)
+    # Precompute masks for each monitor and particle_id using vectorized broadcasting for speed
```

Comment:
Wild that the line below is equivalent to before. Not sure we need to specify we use vectorized blah blah